scripts/prepack.ts

```diff
@@ -96,7 +89,7 @@ void (async () => {
       await runPackagePrepack(packagePrepackPath);
     }
   } catch (error) {
-    console.error(`Error while trying to access ${packagePrepackPath.toString()}`);
+    console.error(`\nERROR: Error while trying to load and run ./scripts/prepack.ts in ${pkgJson.name}:\n${error}`);
```

Comment:
Giga-Nit: I think this would be either slightly more verbose (stack trace) and/or prettier in the console. Not sure though.

```suggestion
    console.error(`\nERROR: Error while trying to load and run ./scripts/prepack.ts in ${pkgJson.name}:`, error);
```

Reply:
I will test it out tomorrow and let you know!

Reply:
You are correct. TIL! Fixed.